projects/kit/components/slider/slider.style.less

```diff
@@ -63,6 +63,14 @@
 
     :not(:disabled)& {
```

Comment:
I wonder if this is required since native disabled state already removes pointer-events.

Reply:
Wow, just checked and it only disabled JavaScript pointer events while CSS :hover state still works.